tools/image_processing/yolo/yolo_prediction.xml

```diff
@@ -0,0 +1,107 @@
+<tool id="yolo_predict" name="Perform YOLO image labeling" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="24.2">
+    <description>with ultralytics</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+    <expand macro="edam" />
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+      export YOLO_CONFIG_DIR='\$HOME/.config/ultralytics' &&
+
+      mkdir -p ./input_images ./runs ./models ./results && 
+      
+      #for $filename in $input_images:
+          ln -s '$filename' './input_images/${filename.element_identifier}' &&
+      #end for
+
+      cp '$class_name' './models/class_name.txt' &&
+
+      cp '$model' './models/model.pt' &&
+		    
+      python '$__tool_directory__/yolov8.py' 
+		    --test_path='./input_images' 
+		    --model_path='./models' 
+		    --model_name='model' 
+		    --run_dir='./runs' 
+		    --save_dir='./results'
+		    --image_size='$image_size'
+		    --mode='$mode'
+		    --foldername='overlaid_images'
+		    --class_names_file='$class_name'
+		    --num_classes=`wc -l < ./models/class_name.txt`
+		    --confidence='$confidence'
+		    --iou='$iou'
+		    --max_det='$max_det'  
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_images" type="data" format="jpg,png,tiff" multiple="true"  label="Input images"/>
+	<param name="class_name" type="data" format="txt" label="YOLO class name file" />
+	<param name="model" type="data" format="data" label="Model file" />
+	<param name="mode" type="select" label="Prediction mode">
+              <option value="segment">segment</option>
+              <option value="detect">detect</option>
+	</param>
+	<param name="image_size" type="integer" value="1000" min="16" max="2048" label="Image size" help="Size of input image to be used only as integer of w,h." />
```

Comment:
I think there should be no upper limit for `image_size`, or have I missed something in the docs? I think it's also good to add a brief comment on what this parameter does:
```suggestion
	<param name="image_size" type="integer" value="1000" min="16" label="Image size" help="All input images will be re-sized to squares with sides of this length (in pixels). This value governs the trade-offs of speed (smaller values) vs accuracy (larger values)." />
```

Reply:
i guess this is galaxy xml recommendation, that all integer float param should have a min, max value.

it can be any value i think